maintainers/maintainer-list.nix

```diff
@@ -718,6 +718,11 @@
     githubId = 79667753;
     keys = [ { fingerprint = "B0D7 2955 235F 6AB5 ACFA  1619 8C7F F5BB 1ADE F191"; } ];
   };
+  ailsa-sun = {
+    name = "Ailsa Sun";
+    email = "jjshenw@gmail.com";
+    github = "ailsa-sun";
```

Comment:
missing `githubId` (Your id is `135079815`, see https://api.github.com/users/ailsa-sun)

Reply:
Ah, thank you. Appreciate it!